### PR TITLE
add filter button

### PR DESCRIPTION
**Pull request by NatalyKokidko**

### CODE REVIEW

**Comment by svetikzp** [comment]
Everything is fine, just plug in the styles css.

File: client/components/filterButton/filterButton.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<link rel="stylesheet" href="./filterButton.css">
```
Comment (TatianaBortnik) [inline client/components/filterButton/filterButton.html:1]: You should copy html from **button** component

File: client/components/filterButton/filterButton.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.filter-btn {
```
Comment (TatianaBortnik) [inline client/components/filterButton/filterButton.css:1]: You should import button.css

**Review by TatianaBortnik — CHANGES_REQUESTED** [review]
You shoul import button.css

**Review by tbazko — CHANGES_REQUESTED** [review]
I'm afraid task was not well refined actually.
You don't need implement design of the button. That is already done within Button component. You needed to make "filtering" UI which should be opened when you click on Filter button. 

There are some designs here, but I'm not sure they fit our purpose. https://www.figma.com/file/lK8ZJBtGidUgLojOiDNkYn/Refund-tracker?node-id=3118%3A359&t=hW64SCszmWr5FwSj-0

@NatalyKokidko I suggest we make this component together. What do you think?

**Comment by NatalyKokidko** [comment]
With pleasure, thanks for your help.

**Pull request closed without merging**